core/server/models/base/index.js

```diff
@@ -79,25 +79,41 @@ ghostBookshelf.Model = ghostBookshelf.Model.extend({
             this.include = _.clone(options.include);
         }
 
-        this.on('creating', this.creating, this);
-        this.on('saving', function onSaving(model, attributes, options) {
-            return Promise.resolve(self.saving(model, attributes, options)).then(function then() {
-                return self.validate(model, attributes, options);
+        ['creating', 'created', 'updated', 'destroying', 'destroyed', 'saved']
```

Comment:
sure can do, just didn't want it to cost any time later when someone tries to use `onUpdating` and can't figure out why it doesn't work 😂